doc/code_data_relocation/relocation.rst

```diff
@@ -0,0 +1,97 @@
+.. _code_data_relocation:
+
+CODE DATA RELOCATION
+####################
+
+Overview
+********
+This script will relocate .text, .rodata, .data and .bss sections from
```

Comment:
@dbkinder I don't understand why there needs to a comma before an "and" in this case. Hence I haven't fixed this.
Others are done. Please re-review.

Reply:
With so many contributors, we've tried to standardize on some simple editorial guidelines such as using English spellings (vs. British) and using the [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/), and [Merriam-Webster Dictionary](http://www.merriam-webster.com/) as guidelines.  Use of the Oxford (or serial) comma is the recommendation, to avoid potential confusion with lists.

Reply:
ohh okay. Changed to the recommendation.
@dbkinder Please re-review.